flow-typed/npm/verdaccio_vx.x.x.js

```diff
@@ -0,0 +1,367 @@
+// flow-typed signature: 70d4cf81a06984155bd79f63db1449a3
+// flow-typed version: <<STUB>>/verdaccio_v^3.1.1/flow_v0.75.0
```

Comment:
You should be able to use  https://github.com/verdaccio/flow-types instead.
```
@verdaccio/types
```

Reply:
Ah, nevermind I just saw it's being used, but ... then I don't get this.

Reply:
You are right, the `@verdaccio/types` are not needed in the flow-typed directory, the import is enough. I'm pushing a correction in a minute.